### ISSUE TITLE
Revisit tests

### ISSUE THREAD

**Issue by AlgusDark** [AlgusDark/bloomer#34]
Need a lot of work in testing, right now the tests have positive cases and lack of some points.

**Comment by MDOR** [comment]
Sup! 
I was looking at the project and I got an idea about set a new structure for the project:
```
\__src
    |__components
    |    |__component_name
    |        |_test   
    |__config
    |    |__tsc
    |    |__jest
    |    |__webpack
    |__setup
        |__tsc
        |__jest
```
We ensure that:
- Each test is contained into the main folder so, tha's easy to understand 
- The config files are not polluting the main file 
- The setup files to do some task are contained in folder so we avoid pollute the root folder

**Comment by AlgusDark** [comment]
Yes, I like that. Let's work that on the new version. Also, we could use jest snapshots.
